sources/api/schnauzer/src/main.rs

```diff
@@ -161,8 +219,7 @@ async fn run() -> Result<()> {
 
     // sundog expects JSON-serialized output so that many types can be represented, allowing the
     // API model to use more accurate types.
-    let output = serde_json::to_string(&setting)
-        .context(error::SerializeOutputSnafu { output: &setting })?;
+    let output = parse_output(setting_type, setting)?;
```

Comment:
This approach doesn't seem correct to me.

The point of `schnauzer` is to render a template, which by definition means we want a string to come out. When given a string value, `serde_json` should serialize it as a string.

I don't really want the dynamic power that's implied by letting `schnauzer` serialize arbitrary templates into arbitrary JSON objects. It's much better for the API model and the generator to cooperate and share a static type definition.

In this case I'd recommend simply not using `schnauzer` at all if its string output is a concern. This is consistent with other generators which emit different types depending on the target setting.

```
bash-5.1# pluto max-pods
58

bash-5.1# netdog node-ip
"10.0.95.193"

bash-5.1# prairiedog generate-boot-settings
{}

bash-5.1# metricdog should-bark
false
```

Reply:
I might be misunderstanding the concern. If we take pluto for example, `max-pods` returns unquoted stdout which the API interprets as a number, but `cluster-dns-ip` is quoted stdout which the API interprets as a string. At the end of the day, the proposed change is really just adding the ability to return a string that is not wrapped in quotes.